### PR TITLE
doc: Clarify RPC versioning

### PR DESCRIPTION
**Pull request by maflcko**

### CODE REVIEW

**Comment by jnewbery** [comment]
ACK fabd4135

**Comment by practicalswift** [comment]
ACK fabd4135c484066c06cb74075f918275b71a7e9e

**Comment by fanquake** [comment]
utACK https://github.com/bitcoin/bitcoin/pull/15799/commits/fabd4135c484066c06cb74075f918275b71a7e9e

File: doc/JSON-RPC-interface.md [hunk 1]
```diff
@@ -5,6 +5,18 @@ The headless daemon `bitcoind` has the JSON-RPC API enabled by default, the GUI
 option. In the GUI it is possible to execute RPC methods in the Debug Console
 Dialog.
 
+## Versioning
+
+The RPC interface might change from one major version of Bitcoin Core to the
+next. This makes the PRC interface implicitly versioned on the major version.
```
Comment (instagibbs) [inline doc/JSON-RPC-interface.md]: `PRC` wow getting political eh?

**Comment by instagibbs** [comment]
utACK except for the part defining interfaces for China

**Comment by practicalswift** [comment]
utACK fabd4135c484066c06cb74075f918275b71a7e9e modulo political content discovered by @instagibbs

**Review by chenlushun — APPROVED** [review]

**Comment by promag** [comment]
utACK

**Comment by fanquake** [comment]
Backported in #15818.

**Pull request merged**